src/readout/main.go

```diff
@@ -124,8 +130,8 @@ func pollSmartPi(config *smartpi.Config, device *i2c.Device) {
 			log.Errorf("Sleep duration negative: %s", sleepFor)
 		} else {
 			log.Debugf("Sleeping for %s", sleepFor)
-			time.Sleep(sleepFor)
```

Comment:
If we are removing this sleep, it makes sense to drop this whole else.  And change the above `Errorf()` to reflect the new timing behavior.